File: Middleware/src/Model/Ride.java
package Model;

import java.util.ArrayList;

public class Ride {

   public String Driver;
   public int Seats;
   public String DeparturePoint;
   public String DestinationCity;
   public String DestinationAddr;
   public String Date;
   public String Time;
   public String Comment;
   public ArrayList<String> passangers;
   


   public Ride(String driver, String Seats) {
      this.Driver = driver;
      this.Seats = Integer.parseInt(Seats);
      passangers = new ArrayList<String>();
   }
   public String getDriver() {
      return Driver;
   }
   
   
   public String getComment()
   {
      return Comment;
   }

   public void setComment(String comment)
   {
      this.Comment = comment;
   }
   
   public String getDeparturePoint()
   {
      return DeparturePoint;
   }

   public void setDeparturePoint(String departurePoint)
   {
      DeparturePoint = departurePoint;
   }
   public String getDestinationCity()
   {
      return DestinationCity;
   }

   public void setDestinationCity(String destCity)
   {
      DestinationCity = destCity;
   }

   public String getDestinationAddr()
   {
      return DestinationAddr;
   }

   public void setDestinationAddr(String destAddr)
   {
      DestinationAddr = destAddr;
   }

   public String getDate()
   {
      return Date;
   }

   public void setDate(String date)
   {
      Date = date;
   }

   public String getTime()
   {
      return Time;
   }

   public void setTime(String time)
   {
      Time = time;
   }
}
  

File: Application/Model/Ride.cs
using System.Collections.Generic;
using Newtonsoft.Json;

public class Ride{

    public Ride(){
        passangers = new List<string>();
    }
    [JsonProperty("Driver")]
    public string Driver { get; set;}

    [JsonProperty("Seats")]
    public int Seats { get; set;}
    [JsonProperty("DeparturePoint")]

    public string DeparturePoint { get; set;}

    [JsonProperty("DestinationCity")]
    public string DestinationCity { get; set;}
    [JsonProperty("DestinationAddr")]

    public string DestinationAddr { get; set;}
    [JsonProperty("Date")]

    public string Date { get; set;}
    [JsonProperty("Time")]

    public string Time { get; set;}
    [JsonProperty("Comment")]
    public string Comment { get; set;}

    [JsonProperty("passangers")]    
    public List<string> passangers { get; set;}
}